config/settings/test.yml

```diff
@@ -449,3 +449,38 @@ accredited_representative_portal:
 vaos:
   eps:
     key_path: modules/health_quest/config/rsa/sandbox_rsa
+
+# Settings for VAProfile
+vet360:
```

Comment:
Do the necessary VA Profile settings exist for local dev and live envs already?

Reply:
My tests wouldn't pass without it. Not sure if it's the new settings configuration.

Reply:
I undid the test.yml changes and seeing if the tests pass now.